### ISSUE TITLE
Custom Geojson Failing

### ISSUE THREAD

**Issue by Murphysc** [metabase/metabase#12597]
Hello,

I am attempting to load a custom map in the Admin panel. However when I click on any of the already saved maps the pages hangs. When I check the logs, it takes a while for the error to show but the logs show the wrong GeoJson endpoint being hit. The endpoint that is being hit is broken becasue of HTTP vs HTTPS, since this causes the modal to freeze it prevents me from update the url to HTTPS. 

However, the real issue is that the custom map I selected is not the endpoint that is throwing an error. Even if an endpoint works, for some reason it's not hitting the right one.

**Logs**
{:message "Connection to host timed out for URL `http://espen.afro.who.int/sites/default/files/content/country/maps/NG-Map.json`",
 :type java.lang.Exception,
 :stacktrace
 ["--> api.geojson$valid_json_url_string_QMARK_.invokeStatic(geojson.clj:63)"
  "api.geojson$valid_json_url_string_QMARK_.invoke(geojson.clj:53)"
  "api.geojson$fn__56258.invokeStatic(geojson.clj:80)"
  "api.geojson$fn__56258.invoke(geojson.clj:79)"
  "api.geojson$validate_custom_geo_json.invokeStatic(geojson.clj:106)"
  "api.geojson$validate_custom_geo_json.invoke(geojson.clj:103)"
  "api.geojson$fn__56277$fn__56278.invoke(geojson.clj:115)"
  "models.setting$set_BANG_.invokeStatic(setting.clj:393)"
  "models.setting$set_BANG_.invoke(setting.clj:378)"
  "api.setting$fn__65013.invokeStatic(setting.clj:34)"
  "api.setting$fn__65013.invoke(setting.clj:28)"
  "middleware.auth$enforce_authentication$fn__66808.invoke(auth.clj:14)"
  "routes$fn__68332$fn__68333.doInvoke(routes.clj:56)"
  "middleware.exceptions$catch_uncaught_exceptions$fn__66907.invoke(exceptions.clj:97)"
  "middleware.exceptions$catch_api_exceptions$fn__66904.invoke(exceptions.clj:85)"
  "middleware.log$log_api_call$fn__68710$fn__68711.invoke(log.clj:197)"
  "middleware.log$log_api_call$fn__68710.invoke(log.clj:191)"
  "middleware.security$add_security_headers$fn__66870.invoke(security.clj:121)"
  "middleware.json$wrap_json_body$fn__68413.invoke(json.clj:64)"
  "middleware.json$wrap_streamed_json_response$fn__68431.invoke(json.clj:100)"
  "middleware.misc$maybe_set_site_url$fn__68740.invoke(misc.clj:59)"
  "middleware.session$bind_current_user$fn__63397$fn__63398.invoke(session.clj:209)"
  "middleware.session$do_with_current_user.invokeStatic(session.clj:184)"
  "middleware.session$do_with_current_user.invoke(session.clj:177)"
  "middleware.session$bind_current_user$fn__63397.invoke(session.clj:208)"
  "middleware.session$wrap_current_user_id$fn__63382.invoke(session.clj:161)"
  "middleware.session$wrap_session_id$fn__63326.invoke(session.clj:125)"
  "middleware.auth$wrap_api_key$fn__66816.invoke(auth.clj:27)"
  "middleware.misc$bind_user_locale$fn__68745.invoke(misc.clj:78)"
  "middleware.misc$add_content_type$fn__68726.invoke(misc.clj:29)"
  "middleware.misc$disable_streaming_buffering$fn__68754.invoke(misc.clj:94)"
  "server$async_proxy_handler$fn__68488.invoke(server.clj:70)"
  "server.proxy$org.eclipse.jetty.server.handler.AbstractHandler$ff19274a.handle(Unknown Source)"]}

**To Reproduce**
1. Go to 'Settings'
2. Click on 'Admin'
3. Click on 'Maps'
4. Select any Custom Map 

**Screenshots**
![image](https://user-images.githubusercontent.com/7292150/83043393-d4e63900-a010-11ea-8050-6b6c8abd46a2.png)

![image](https://user-images.githubusercontent.com/7292150/83043402-d9aaed00-a010-11ea-9ad7-bb831935d219.png)

**Information about Metabase Installation:**

- Browser: Google Chrome 83.0.4103.61
- Operating system:  OS X 10.15
- Your databases: MySQL
- Metabase version: 0.35.3
- Metabase hosting environment: Jar-file
- Metabase internal database: MySQL

**Comment by flamber** [comment]
Hi @Murphysc 
I guess WHO requires https - try this instead, it works:
`https://espen.afro.who.int/sites/default/files/content/country/maps/NG-Map.json`
Please use the forum for questions and troubleshooting: https://discourse.metabase.com/

**Comment by Murphysc** [comment]
I'm not able to update the URL though. I am attempting to find the urls in the DB but not luck. As i stated the issue is the wrong url is being it. The one that is erroring is not the map I am tyring to load, so it shouldn't even be showing that endpoint.

**Comment by flamber** [comment]
@Murphysc Okay, then there should be errors in your browser console.
Can you provide steps to reproduce?

**Comment by Murphysc** [comment]
@flamber the steps to reproduce are included in the original post. I suppose the key is to have a broken URL.

I've included a screenshot of the error in the browser. It's pretty useless which is why I checked our logs on the server. This pointed to a URL timeout error but as mentioned the URL that is throwing the error is not the url related to the MAP I am attempting to update.

![image](https://user-images.githubusercontent.com/7292150/83046124-8c307f00-a014-11ea-9169-f4abd2a0fe6f.png)
